platform_channels/test/src/event_channel_demo_test.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:platform_channels/src/event_channel_demo.dart';
+
+void main() {
+  group('EventChannel Demo tests', () {
+    testWidgets('EventChannel accelerometer Stream test', (tester) async {
+      await tester.pumpWidget(MaterialApp(
+        home: EventChannelDemo(),
+      ));
+
+      expect(find.text('No Data Available'), findsOneWidget);
+    });
```

Comment:
I'm not sure how can I mock the `EventChannel.recieveBroadcastStream`. Suggestions and ideas for testing are much appreciated. I can surely write a unit test, but it won't be able to test the widgets.

Reply:
The `EventChannel` test in the framework help me. 

[Here](https://github.com/flutter/flutter/blob/7519c383f9bcbe54a235ce4b036bdefb99a3ed2d/packages/flutter/test/services/platform_channel_test.dart#L275)